src/assets/sass/protocol/components/_card.scss

```
@@ -134,6 +134,24 @@
     }
 }
 
+//* -------------------------------------------------------------------------- */
+// Dark mode styling
+
+.mzp-c-card.mzp-t-dark {
```

We should also add a selector that lets cards in a dark container take on the theme:
```suggestion
.mzp-c-card.mzp-t-dark,
.mzp-t-dark .mzp-c-card {
```